README.md

```diff
@@ -11,26 +11,30 @@
 ## Introduction
 [//]: # ([The “executive summary” or “abstract”. Explain in a few sentences what the goals of the project are, and a brief overview of how the solution works. This should be no more than 1-2 paragraphs.])
 
-With the advent of Virtual and Augmented reality, a new style of 3D interfaces is emerging. We are no longer limited to flat screens; instead the user interface can be broken up in sections and positioned within your 3D space (for VR) or the real world (for AR).
+With the advent of virtual and augmented reality, a new style of 3D interfaces is emerging.
+We are no longer limited to flat screens; instead the user interface can be broken up in sections and positioned within the user's 3D space (for VR) or the real world (for AR).
 
 Browser vendors are developing WebXR for WebGL rendered content and are experimenting with [basic 3D models](https://developers.google.com/web/updates/2019/02/model-viewer) and other [interactive experiences](https://creator.magicleap.com/learn/guides/prismatic-getting-started).
-However, until now it was not possible to position CSS generated content in 3D.
+However, until now, positioning CSS generated content in 3D was not possible.
 
-## proposal
+## Proposal
 
-Our goal is to break off parts of web page into the 3-d space around the browser surface. 
+Our goal is to break off parts of web page into the 3D space around the browser surface. 
 
-This would make viewing web pages a much more immersive experience rather than looking at a rectangular surface. We hope to do accomplish this by introducing a small addition to CSS.
+This would make viewing web pages a much more immersive experience rather than looking at a rectangular surface.
+We hope to accomplish this by introducing a small addition to CSS.
 
 We propose to introduce a new value for `transform-style`: `detached`.
 
-This new property builds upon the existing 3D transforms that are shipping in all browsers and extends the behavior of `transform-style: preserve-3d`. The main difference is that instead of flattening the transformed elements back to the page, the transformed element stays in 3D space.
+This new property builds upon the existing 3D transforms that are shipping in all browsers and extends the behavior of `transform-style: preserve-3d`.
+The main difference is that instead of flattening the transformed elements back to the page, the transformed element stays in 3D space.
 If `transform-style: detached` is specified on an element, children with 3D transforms will be lifted out of the page.
 
 Here is an example rendering of the effect:
```

Comment:
Please give credits to original owner. (Josh Carpenter)

Reply:
will include in the commented out section 'references and acknowledgements' in the end